packages/ee/project-members/src/lib/utils.ts

```diff
@@ -1,7 +1,8 @@
-import { ProjectMemberRole } from '@activepieces/ee-shared';
+import { ProjectMemberRole } from '@activepieces/shared';
 
 export const RolesDisplayNames: { [k: string]: string } = {
   [ProjectMemberRole.ADMIN]: $localize`Admin`,
   [ProjectMemberRole.EDITOR]: $localize`Editor`,
   [ProjectMemberRole.VIEWER]: $localize`Viewer`,
+  [ProjectMemberRole.EXTERNAL_CUSTOMER]: $localize`External Customer`,
```

Comment:
Can you add issue to hide it when feature is not enabled

Reply:
I will add it in front end.